lib/sprockets/sassc_processor.rb

```diff
@@ -24,7 +24,7 @@ def call(input)
       engine = Autoload::SassC::Engine.new(input[:data], options)
 
       css = Utils.module_include(Autoload::SassC::Script::Functions, @functions) do
-        engine.render
+        engine.render.gsub(/^\/\*# sourceMappingURL=.*\*\/$/, '').strip
```

Comment:
The solves the `omit_source_map_url` problem until the option makes into the next sassc version.